pkg/backup/restorer/restorer.go

```diff
@@ -101,8 +102,16 @@ func (br *BackupRestorer) load() error {
 		}
 	}()
 
-	if err := os.Remove(fmt.Sprintf("%s/%s", filepath.Dir(br.dbPath), filepath.Base(br.dbPath))); err != nil {
-		log.Warn().Err(err).Msg("removing database file")
+	dbPath := fmt.Sprintf("%s/%s", filepath.Dir(br.dbPath), filepath.Base(br.dbPath))
+	tmpDbPath := dbPath + ".tmp"
+
+	// If a database already exists we are going to rename it,
+	// so, later, we can get the node id information from the existing database.
+	dbExists := fileExists(dbPath)
+	if dbExists {
+		if err := os.Rename(dbPath, tmpDbPath); err != nil {
+			log.Warn().Err(err).Msg("renaming file")
+		}
```

Comment:
Maybe it's safer to return an error and let the operator figure out why this failed.
If the renaming didn't work correctly, I've a sense something somewhat bad could happen if we continue.